tips/TIP-0025/core-rest-api.yaml

```diff
@@ -0,0 +1,3329 @@
+              properties:
+                index:
+                  type: integer
+                  description: The index of the latest seen milestone.
+                timestamp:
+                  type: integer
+                  description: The timestamp of the latest seen milestone.
+                milestoneId:
+                  type: string
+                  description: The Milestone ID of the latest seen milestone. Hex-encoded with 0x prefix.
+              required:
+                - index
+            confirmedMilestone:
+              type: object
+              description: Information about the latest confirmed milestone.
+              properties:
+                index:
+                  type: integer
+                  description: The index of the latest confirmed milestone.
+                timestamp:
+                  type: integer
+                  description: The timestamp of the latest confirmed milestone.
+                milestoneId:
+                  type: string
+                  description: The Milestone ID of the latest confirmed milestone. Hex-encoded with 0x prefix.
+              required:
+                - index
+          required:
+            - isHealthy
+            - latestMilestone
+            - confirmedMilestone
+        metrics:
+          description: Node metrics.
+          properties:
+            blocksPerSecond:
+              description: The current rate of new blocks per second.
+              type: number
+              format: float
+            referencedBlocksPerSecond:
+              description: The current rate of referenced blocks per second.
+              type: number
+              format: float
+            referencedRate:
+              description: The ratio of referenced blocks in relation to new blocks of the last confirmed milestone.
+              type: number
+              format: float
+          required:
+            - blocksPerSecond
+            - referencedBlocksPerSecond
+            - referencedRate
+        supportedProtocolVersions:
+          type: array
+          description: The supported protocol versions.
+          items:
+            type: integer
+        protocol:
+          description: Protocol parameters.
+          properties:
+            networkName:
+              type: string
+              description: The Name of the network from which the networkId is derived.
+            bech32HRP:
+              type: string
+              description: Tells whether the node supports mainnet or testnet addresses. Value `iota` indicates that the node supports mainnet addresses. Value `atoi` indicates that the node supports testnet addresses.
+            tokenSupply:
+              type: string
+              description: Current supply of base token. Plain string encoded number.
+            protocolVersion:
+              type: integer
+              description: Protocol version used by the network.
+            minPoWScore:
```

Comment:
This is not proper camelCase format (it translates to `min-Po-W-Score`). We should change it to `minPOWScore` or `minPowScore`. I prefer the later for readability.

The problem with using inconsititent formatting is that tools that automatically generate code struggle with the current format. The same thing happens in INX by the way.

Reply:
Changed in https://github.com/iotaledger/tips/pull/57/commits/103cdfa643012c0120faf4088cf5cd745e1b5de2